core/iwasm/common/wasm_runtime_common.c

```diff
@@ -554,6 +584,46 @@ wasm_runtime_full_init(RuntimeInitArgs *init_args)
     return true;
 }
 
+bool
+wasm_runtime_is_running_mode_supported(RunningMode running_mode)
+{
+    if (running_mode == 0) {
```

Comment:
what's 0?

Reply:
if it means the default, i guess it's better to have `Mode_Default` instead.

Reply:
Yes, `Mode_Default` is more readable. The reason for using a 0 rather than adding a `Mode_Default` in the enum type is that I think it's maybe confusing for the user to use the 'default' running mode (for a different compilation setup, the default may mean a different actual running mode). Maybe I can add an internal macro/enum, `Mode_Default` to prevent users from using it and make it more readable; what do you think?